### PR TITLE
shalinee | stage | PO responsive code cherrypick

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html [hunk 1]
```diff
@@ -213,11 +213,17 @@
                         <td [attr.data-title]="localeData?.vendor_name">
                             {{item.vendor?.name}}
                         </td>
+<<<<<<< HEAD
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html:216]: Improper merge conflict resolution

File: apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html [hunk 2]
```diff
@@ -213,11 +213,17 @@
...
                         <td [attr.data-title]="localeData?.total_amount" [tooltip]="grandTotalPopupTemplate"
                             container="body">
                             <span class="d-flex justify-content-end">
                                 <amount-field [amount]="item.grandTotal?.amountForAccount"
                                     [currencySymbol]="item.grandTotal?.currencyForAccount?.symbol"
+=======
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html:222]: Improper merge conflict resolution

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/3thkz3
cherry-picked PO responsive code
@shalinee-walkover

File: apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html]: Solved

File: apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/purchase/purchase-order/purchase-order.component.html]: Solved

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**